charts/datadog/templates/_kubernetes_apiserver_config.yaml

```diff
@@ -0,0 +1,10 @@
+{{- define "kubernetes_apiserver-config" -}}
+{{- if and .Values.datadog.collectEvents .Values.datadog.kubernetesEvents.unbundleEvents -}}
```

Comment:
why not or?

Reply:
because `unbundleEvents` is a sub option of "even collection" feature.

`.Values.datadog.collectEvents` is use also to create the corresponding RBAC... 

But indeed, we can change a bit the logic to enable `collectEvents` when `unbundleEvents` is set to true